lib/phplint.js

```diff
@@ -14,7 +14,7 @@ function spawn() {
 function filter(error, err) {
   const filteredStream = new Stream.PassThrough({ objectMode: true });
 
-  filteredStream._transform = (file, unused, done) => { // eslint-disable-line no-underscore-dangle
```

Comment:
arrow functions bind to the current `this` context by default. In this location, `this` is the `filter()` function and not the `filteredStream`.